src/content/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.mdx

```diff
@@ -188,744 +188,70 @@ The following are proposed time ranges for EOL on older Node.js versions. The ac
-        </td>
-      </tr>
-
-      <tr>
-        <td>
-          [Koa](https://www.npmjs.com/package/koa)
-        </td>
-
-        <td>`koa`</td>
-
-        <td>
-          2.0.0
-        </td>
-
-        <td>3.2.0</td>
-
-        <td>
-            [External module](https://github.com/newrelic/node-newrelic-koa) loaded with the agent
-        </td>
-      </tr>
-
-      <tr>
-        <td>
-          [Fastify](https://www.npmjs.com/package/fastify)
-        </td>
-
-        <td>`fastify`</td>
-
-        <td>
-          2.0.0
-        </td>
-
-        <td>
-          8.5.0
-        </td>
-
-        <td>
-        </td>
-      </tr>
-
-      <tr>
-        <td>
-          [Nest.js](https://nestjs.com/)
-        </td>
-
-        <td>`@nestjs/core`</td>
-
-        <td>
-          8.0.0
-        </td>
-
-        <td>
-          10.1.0
-        </td>
-
-        <td>
-          If using the `nest start` command to start the application, modify its startup binary to load the New Relic agent: `nest start --exec 'node -r newrelic'`
-        </td>
-      </tr>
-
-      <tr>
-        <td>
-          [Next.js](https://www.npmjs.com/package/next)
-        </td>
-
-        <td>`next`</td>
-
-        <td>
-          12.0.9 - 13.3.0
-        </td>
-
-        <td>
-          10.1.0
-        </td>
-
-        <td>
-          v12.2.0 or higher required for middleware instrumentation
-        </td>
-      </tr>
-
-    </tbody>
-  </table>
-
-If you are using a supported framework with default routers, the Node.js agent can read these frameworks' route names as is. However, if you want more specific names than are provided by your framework, you may want to use one or more of the tools New Relic provides with the [Node.js transaction naming API](/docs/nodejs/nodejs-transaction-naming-api).
-
-  <Callout title="EOL NOTICE">
-    We've discontinued support for several capabilities in November 2021. This includes the Oracle Driver Package and Hapi versions prior to Hapi 19.2 for our Node.js agent. For more details, including how you can prepare for this transition, see our [Support Forum post](https://discuss.newrelic.com/t/important-upcoming-changes-to-capabilities-and-support-across-node-agents-suggested-pagerduty-responders-incident-workflows-and-kubernetes-instrumentation/164481?u=dholloran).
-  </Callout>
-</Collapser>
-</CollapserGroup>
+{/* begin: compat-table */}
+
+## Instrumented Modules 
+
+After installation, the agent automatically instruments with our catalog of supported Node.js libraries and frameworks. This gives you immediate access to granular information specific to your web apps and servers.  For unsupported frameworks or libraries, you'll need to instrument the agent yourself using the [Node.js agent API](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/api-guides/nodejs-agent-api/).
```

Comment:
```suggestion
After installation, our agent automatically instruments with a catalog of supported Node.js libraries and frameworks. This gives you immediate access to granular information specific to your web apps and servers. For unsupported frameworks or libraries, you'll need to instrument the agent yourself using the [Node.js agent API](/docs/apm/agents/nodejs-agent/api-guides/nodejs-agent-api/).
```

Reply:
As stated in the other review comment, all of this text is also replicated in the doc included in our agent repository. So is it a definite requirement that URL be relative instead of absolute?

Reply:
Sure, we can include the absolute link in this situation, similar to exceptions we make for automatically generated release notes.